src/main/java/domain/LottoOrder.java

```diff
@@ -0,0 +1,49 @@
+package domain;
+
+import exception.ExceptionMessage;
+
+public class LottoOrder {
+    private final int manualTicketCount;
+    private final int autoTicketCount;
+
+    public LottoOrder(Payment payment, int manualTicketCount, int autoTicketCount) {
+        validateLottoOrder(payment.calculateLottoCount(), manualTicketCount, autoTicketCount);
+        this.manualTicketCount = manualTicketCount;
+        this.autoTicketCount = autoTicketCount;
+    }
+
+    private static void validateLottoOrder(int totalLottoTicketCount, int manualTicketCount, int autoTicketCount) {
+        validateTicketCount(manualTicketCount);
+        validateTicketCount(autoTicketCount);
+        validateManualTicketCount(totalLottoTicketCount, manualTicketCount);
+        validateTotalTicketCount(totalLottoTicketCount, manualTicketCount, autoTicketCount);
+    }
+
+    private static void validateTicketCount(int ticketCount) {
+        if (ticketCount < 0) {
+            throw new IllegalArgumentException(ExceptionMessage.INVALID_TICKET_COUNT.getMessage());
+        }
+    }
+
+    private static void validateManualTicketCount(int totalLottoTicketCount, int manualTicketCount) {
+        if (totalLottoTicketCount < manualTicketCount) {
+            throw new IllegalArgumentException(ExceptionMessage.INVALID_MANUAL_TICKET_COUNT.getMessage());
```

Comment:
이건 개인적인 취향인데요. 
```suggestion
            throw new IllegalArgumentException(INVALID_MANUAL_TICKET_COUNT.getMessage());
```
이런식으로 작성해도 좋을거 같기도 해요 :) ㅎㅎㅎㅎ
어차피 static이니, 이렇게 쓸수가 있다는것 정도만 참고해주세요!